src/main/kotlin/org/wfanet/measurement/kingdom/db/testing/Databases.kt

```diff
@@ -0,0 +1,12 @@
+package org.wfanet.measurement.kingdom.db.testing
+
+import org.wfanet.measurement.kingdom.db.LegacySchedulingDatabase
+import org.wfanet.measurement.kingdom.db.ReportDatabase
+import org.wfanet.measurement.kingdom.db.RequisitionDatabase
+
+data class Databases(
```

Comment:
curious why one is called Databases.kt and one is called KingdomDatabases.kt (under /spanner/).